api-common/src/main/kotlin/io/qalipsis/api/meters/Gauge.kt

```diff
@@ -27,7 +29,27 @@ interface Gauge : Meter<Gauge> {
      *
      * @return The current value.
      */
-    fun value(): Double {
-        return Double.NaN
-    }
+    fun value(): AtomicInteger
+
+    /**
+     * Update the gauge by one.
+     */
+    fun increment(): AtomicInteger
```

Comment:
All the functions should return a double (a normal one, not atomic), since we manage the value as a double (the argument is a double)